docs/getting-started/steps-to-build-deploy-contract.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Steps to build and deploy a smart contract
+sidebar_position: 8
+---
+
+This is an explanation of the process of building a smart contract in Marlowe from start to deployment on the preview or pre-production network of the Cardano blockchain. 
+
+## Building a smart contract
+
+Creating a smart contract involves the following stages: 
+* Conceptualization
+* Design
+* Development
+* Simulation and testing
+
+### Conceptualization
+
+Take the time to think through what exactly you want your smart contract to accomplish. Carefully consider what the key advantages would be of conducting your transaction with a smart contract on the blockchain. Here are some essential questions to ask yourself as you conceptualize your contract: 
+* What is the central purpose of your smart contract? For example, it could be to facilitate a transaction or to enforce an agreement. 
+* Who are your participants? 
+* What roles and responsibilities will each of them they have? 
+* What conditions need to be met along the way in order for the contract to proceed through each stage? 
+* Think about the logic that the contract will need to take into account in the scenario where one or more participants do not fulfill their obligations. How will you protect your interests? 
+* What conditions need to be part of the contract so that each participant is assured that the terms are fair and reasonable? 
+* Will you need to take into account any external data or events? If so, how will you handle those factors? 
+
+### Design
+
+After completing the conceptualization stage, you can start designing your smart contract. 
+
+- **Structure**: Layout the structure of your contract. Think about the sequence of actions that will be involved. How will funds need to be handled at each stage? 
```

Comment:
Please change all the colons to full stops for consistency with the writing guidelines.